packages/typespec-client-generator-core/test/decorators/general-list.test.ts

```diff
@@ -338,11 +338,17 @@ describe("csharp only decorator", () => {
   it("@useSystemTextJsonConverter", async function () {
     runner = await createSdkTestRunner(
       {},
-      { additionalDecorators: ["Azure\\.ClientGenerator\\.Core\\.@useSystemTextJsonConverter"] },
+      {
```

Comment:
The test description is for another decorator.
We should separate it to a isolated test if we want to make sure the e2e works.

Reply:
refined the tests.